pathml/preprocessing/transforms.py

```diff
@@ -1415,14 +1419,30 @@ def __init__(self, segmentation_mask):
     def __repr__(self):
         return f"QuantifyMIF(segmentation_mask={self.segmentation_mask})"
 
-    def F(self, tile):
-        # pass (x, y, channel) image and (x, y) segmentation
-        img = tile.image.copy()
-        segmentation = tile.masks[self.segmentation_mask][:, :, 0]
+    def F(self, img, segmentation, coords_offset=(0, 0)):
```

Comment:
This is a preference think but using type hinting can help a ton especially when referring to the code later on. Consider a doing something like 
```suggestion
    def F(self, img: np.ndarray, segmentation: np.ndarray, coords_offset: tuple=(0, 0)) -> np.ndarray:
```